cpp/src/neighbors/nn_descent_c.cpp

```diff
@@ -71,6 +71,57 @@ void* _build(cuvsResources_t res,
     RAFT_FAIL("dataset must be accessible on host or device memory");
   }
 }
+
+template <typename output_mdspan_type>
+void _get_graph(cuvsResources_t res, cuvsNNDescentIndex_t index, DLManagedTensor* graph)
+{
+  auto dtype = index->dtype;
+  if ((dtype.code == kDLUInt) && (dtype.bits == 32)) {
+    auto index_ptr = reinterpret_cast<cuvs::neighbors::nn_descent::index<uint32_t>*>(index->addr);
+    auto dst       = cuvs::core::from_dlpack<output_mdspan_type>(graph);
+    auto src       = index_ptr->graph();
+    auto res_ptr   = reinterpret_cast<raft::resources*>(res);
+
+    RAFT_EXPECTS(src.extent(0) == dst.extent(0), "Output graph has incorrect number of rows");
+    RAFT_EXPECTS(src.extent(1) == dst.extent(1), "Output graph has incorrect number of cols");
+
+    cudaMemcpyAsync(dst.data_handle(),
+                    src.data_handle(),
+                    dst.extent(0) * dst.extent(1) * sizeof(float),
```

Comment:
I think this should be the type of indices (`sizeof(uint32_t)`) instead of `sizeof(float)`?

Reply:
good catch! I fixed in the latest commit